# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1380136594 14400
#      Wed Sep 25 15:16:34 2013 -0400
# Node ID fb5bdabb9169c8ca8fd03dc6929a6b0d2b4454ec
# Parent  278ef6bd821d43da42e88a355d34fdbae2d3827f
svds.m: Increase tolerance in test.

diff --git a/scripts/sparse/svds.m b/scripts/sparse/svds.m
--- a/scripts/sparse/svds.m
+++ b/scripts/sparse/svds.m
@@ -292,15 +292,15 @@ endfunction
 %!testif HAVE_ARPACK
 %! [u2,s2,v2,flag] = svds (zeros (10), k);
 %! assert (u2, eye (10, k));
 %! assert (s2, zeros (k));
 %! assert (v2, eye (10, 7));
 %!
 %!testif HAVE_ARPACK
 %! s = svds (speye (10));
-%! assert (s, ones (6, 1), 4*eps);
+%! assert (s, ones (6, 1), 8*eps);
 
 %!test
 %! ## Restore random number generator seeds at end of tests
 %! rand ("state", rand_state);
 %! randn ("state", randn_state);
 
